content/push/device.textile

```diff
@@ -53,6 +54,11 @@ h3. Configure APNs for iOS devices
 ** Copy everything between and including @-----BEGIN CERTIFICATE-----@ and @-----END CERTIFICATE-----@, and paste it into the *PEM cert* text box of the Apple push notification service section of your Ably notifications app "dashboard":https://ably.com/accounts.
 ** Similarly, copy everything between and including @-----BEGIN PRIVATE KEY-----@ and @-----END PRIVATE KEY-----@, and paste it into the *PEM private key* text box of the same section. Then, click *save*.
 
+h3. Configure a Web Push device
+
+* Web push is supported using the "W3C Push API":https://www.w3.org/TR/push-api/ using an Ably provided VAPID key.
```

Comment:
yeah unlike FCM and APNs you don't really need to do any app/account level configuration for web push (at least for now, we may add some additional configuration later on down the line), so I wanted to use this section to highlight:
1. the fact that you don't need to do any configuration because we provide our own VAPID key pair
2. the fact that you need to host a service worker which is available to clients (i guess you could argue that this is something you need to do to 'configure' web push)

let me know if you think there's a better place for that information in this doc

Reply:
Let's use the above as "steps" then for the config section. Thank you for clarifying!